zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -115,6 +115,11 @@ object ServerSpec extends HttpRunnableSpec(8088) {
         val path = getClass.getResource("/TestFile").getPath
         val res  = Http.data(HttpData.fromStream(ZStream.fromFile(Paths.get(path)))).requestBodyAsString()
         assertM(res)(containsString("foo"))
+      } +
+      testM("Response Content length for Unicode Charset bytes") {
+        val app           = Http.text("åäö")
```

Comment:
Use a Gen to generate Unicode values.

Reply:
Done changed to anyString which internally use anyUnicodeChar

```
  def anyString: Gen[Random with Sized, String] =
    Gen.string(Gen.anyUnicodeChar)
```